templates/fsharp/templatedcontrol/NewTemplatedControl.axaml.fs

```diff
@@ -0,0 +1,8 @@
+namespace AvaloniaAppTemplate.Namespace
+
+open Avalonia
+open Avalonia.Controls
```

Comment:
We don't need these two usings here,if it is copy-pasta from C# feel free to remove that from C# version also

Reply:
Yes, that was copy pasta, will remove them from both

Reply:
done